Tests/BaseLib/excludeObjectCopy.cpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * \date   2015-03-18
+ * \brief
+ *
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include <algorithm>
+#include <cstddef>
+#include <vector>
+
+#include "gtest/gtest.h"
+
+#include "BaseLib/excludeObjectCopy.h"
+
+TEST(BaseLib, excludeObjectCopy)
+{
+	// create and fill vector
+	std::size_t const size(100);
+	std::vector<std::size_t> v(size);
+	std::iota(v.begin(), v.end(), 0);
+
+	std::vector<std::size_t> ex_positions(size/10);
+	// do not copy first 10 elements
+	std::iota(ex_positions.begin(), ex_positions.end(), 0);
+
+	std::vector<std::size_t> c1(BaseLib::excludeObjectCopy(v,ex_positions));
+	ASSERT_EQ(size-ex_positions.size(), c1.size());
```

Comment:
Is this a typo? I can not find the location where the variable `size-ex_positions` gets declared / defined.